### ISSUE TITLE
Doesn't output DECtalk phonetics

### ISSUE THREAD

**Issue by scruss** [GarettMorrison/DECTALK_Choir#2]
It would be really useful if each channel's phonetic text were output somewhere. The video and audio are neat and all, but the real gold is in the phonemes.

**Comment by Will-Morr** [comment]
The phonemes are output line by line in outputs/*SongName*/*TrackName* in Dectalk readable format. The functions that do this conversion are in pyFuncs/PhonemeProcessing.py. You can call lyricsToPhonemes() with just the filename and get the phonemes line by line. I'd be happy include the raw phoneme output somewhere, but I'm not sure what format would actually be useful. A straight text dump is super hard to parse visually, its how I debugged early versions and it sucked.
